internal/pkg/cli/svc_logs.go

```diff
@@ -193,6 +209,31 @@ func (o *svcLogsOpts) Execute() error {
 	return nil
 }
 
+func (o *svcLogsOpts) setPreviousStoppedTaskID() error {
+	env, err := o.configStore.GetEnvironment(o.appName, o.envName)
+	if err != nil {
+		return fmt.Errorf("get environment %s: %w", o.envName, err)
+	}
+	sess, err := o.sessProvider.FromRole(env.ManagerRoleARN, env.Region)
+	if err != nil {
+		return fmt.Errorf("get session %s: %w", o.envName, err)
+	}
+	svcDesc, err := o.newSvcDescriber(sess).DescribeService(o.appName, o.envName, o.name)
+	if err != nil {
+		return fmt.Errorf("describe serivce %s: %w", o.name, err)
+	}
+	if len(svcDesc.StoppedTasks) > 0 {
+		taskID, err := awsecs.TaskID(aws.StringValue(svcDesc.StoppedTasks[0].TaskArn))
+		if err != nil {
+			return err
+		}
+		o.taskIDs = append(o.taskIDs, taskID)
```

Comment:
What do you think isntead of appending we just set to the single value?
```suggestion
		o.taskIDs = []string{ taskID }
```

Reply:
What do you think of returning an error in `Validate` if the user specificies both `--tasks` and `--previous`?